docs/ipdk-dpdk.md

```diff
@@ -9,20 +9,20 @@ This document describes how to build and run IPDK networking recipe on DPDK targ
 ### Build P4 SDE for DPDK
 
 ```bash
-git clone --recursive git@github.com:p4lang/p4-dpdk-target.git p4sde
+   git clone --recursive git@github.com:p4lang/p4-dpdk-target.git p4sde
 ```
 
 For build instructions, refer [P4 SDE Readme](https://github.com/p4lang/p4-dpdk-target/blob/main/README.md#building-and-installing)
 
 ### Build and install infrap4d dependencies
 
 ```bash
-git clone --recursive git@github.com:ipdk-io/networking-recipe.git ipdk.recipe
-cd ipdk.recipe
-export IPDK_RECIPE=`pwd`
-cd $IPDK_RECIPE/setup
-cmake -B build -DCMAKE_INSTALL_PREFIX=<dependency install path> [-DUSE_SUDO=ON]
-cmake --build build [-j<njobs>]
+   git clone --recursive git@github.com:ipdk-io/networking-recipe.git ipdk.recipe
```

Comment:
You generally don't need to indent text in a code block. The change of font plus the gray backdrop make the display text stand out without need for further emphasis.